### PR TITLE
feat: support on-prem environment

### PR DESCRIPTION
**Pull request by GeunjeLEE**
### Task
- [x] New Guide
- [ ] Fix Content
- [ ] Etc (CI/CD Workflow)

### Description
Helm chart 1.10.5 has been added for on-premises environments.

### CODE REVIEW

File: examples/v1.10.5/on-premise/additional/eco/docs.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          targetPort: 80
+          protocol: TCP
+
+ingress:
+    enabled: false
+
+global:
+    namespace: {name_space}
+    serviceAccountName: {service_account_name}
+    common_env:
```
Comment (whdalsrnt) [inline examples/v1.10.5/on-premise/additional/eco/docs.yaml:44]: The docs service is a static web page and requires no internal communication.
Therefore, it seems that you can remove all PROXY settings.

**Review by whdalsrnt — CHANGES_REQUESTED** [review]
Thank you for your PR.
It seems that the proxy settings of SpaceONE, which is currently running in the on-premise environment, need to be applied only to the plguins.

File: examples/v1.10.5/on-premise/additional/eco/docs.yaml (hunk 1 above)
Comment (GeunjeLEE) [reply examples/v1.10.5/on-premise/additional/eco/docs.yaml:44]: Thank you for your feedback.
I agree with your feed, so I'll update this example.

**Review by whdalsrnt — COMMENTED** [review]
Please check the following spacectl related reviews.

File: examples/v1.10.5/on-premise/additional/eco/spacectl.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        memory: "256Mi"
+        cpu: "50m"
+
+imagePullSecrets: 
+  - name: {secret_name_for_credential}
+
+serviceAccountName: {service_account_name}
+
+env:
+  - name: http_proxy
```
[the remark this answers is not in the document]
Comment (GeunjeLEE) [reply examples/v1.10.5/on-premise/additional/eco/spacectl.yaml:23]: But spacectl needs to access pypi sometimes so spacectl needs this configuration.

[the remark this answers is not in the document]
Comment (whdalsrnt) [reply examples/v1.10.5/on-premise/additional/eco/spacectl.yaml:23]: OK, I understand.
Thank you!

**Pull request merged**